### PR TITLE
[cling-cpt] Added build option for the platform tags to the argument parser [skip-ci]

### PR DESCRIPTION
**Pull request by saisoma123**
# This Pull request: Adds another build option for building Cling with specific platform tags. 

## Checklist:

- [X] tested changes locally
- [NA] updated the docs (if necessary)

This PR fixes #406 mentioned in (https://github.com/root-project/cling/issues/406)

### CODE REVIEW

File: interpreter/cling/tools/packaging/cpt.py [hunk 1]
```diff
@@ -2375,7 +2383,8 @@ def custom_input(prompt, always_yes=False):
         tarball()
         cleanup()
 
-if args['tarball_tag']:
+if bool(args['tarball_tag']) ^ bool(args['tarball_tag_build']):
```
Comment (vgvassilev) [inline interpreter/cling/tools/packaging/cpt.py:2386]: Can `^` be replaced by `or`?

Comment (saisoma123) [reply interpreter/cling/tools/packaging/cpt.py:2386]: I wanted to use an exclusive or so that two of the options couldn't be called at the same time

Comment (vgvassilev) [reply interpreter/cling/tools/packaging/cpt.py:2386]: I would prefer to use or and make this check before we even come here. We can check and diagnose this condition where the user will know what they did wrong…

Comment (saisoma123) [reply interpreter/cling/tools/packaging/cpt.py]: Hi @vgvassilev just added the changes like you said

**Review by vgvassilev — APPROVED** [review]

**Pull request merged**